### ISSUE TITLE
v3.10.1 _.sortByOrder [orders] specifies string[] but appears to need falsy value to sort 'desc'

### ISSUE THREAD

**Issue by danzaner** [lodash/lodash#1702]
The v3.10.1 docs for the [orders] argument to _.sortByOrder specifies a string array (and the examples use strings), but I'm finding that I can't get a descending sort unless I set the 'orders' array value corresponding to my field name to a boolean 'false'. Setting the order array value to the string 'desc' appears to result in an ascending sort because the value is truthy. The build used was downloaded today from: https://raw.github.com/lodash/lodash/3.10.1/lodash.js

**Comment by jdalton** [comment]
Can you provide an example.

**Comment by danzaner** [comment]
Well, providing an example is turning out to be harder than I thought it would be.

I started with a JSFiddle using cdnjs.com for lodash and some real data from my app. That sorted correctly, which surprised me, so I went back to my Angular app and switched:

    var sortedComments = _.sortByOrder(tempComments, ['created_at'], [false]);

back to what I originally tried:

    var sortedComments = _.sortByOrder(tempComments, ['created_at'], ['desc']);

That fails to sort descending in my app every time.

Reproducing this in a fiddle is not that straightforward, though. _.sortByOrder is getting called in a function being called from an Angular ng-click handler on a directive, and I'm now inclined to think this is maybe an issue of loading an older version of lodash somewhere in a dependency of my Angular app. I found a reference to using truthy/falsy values with _.sortByOrder() here: https://leanpub.com/lodashcookbook/read , so I can only assume that older versions of the lodash library intentionally used them. I don't think I want to figure out how many different libraries I'm using that also depend on lodash -- it's probably quite a few. I think it's safe to assume that 3.10.1 is working fine, and that an eventual update in my app's dependencies will fix my issue.

Thanks for your interest and quick response. Lodash has been incredibly useful for me for a long time. I really appreciate all the work you and the other contributors have done!

-- Dan
